pootle/apps/virtualfolder/urls.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.conf.urls import include, url
+
+from pootle_store.urls import unit_xhr_urlpatterns
+
+from .views import get_vfolder_units, VFolderTPTranslateView
+
+
+vfolder_urlpatterns = [
+    # TP Translate
+    url(r'(?P<language_code>[^/]*)/'
+        r'(?P<project_code>[^/]*)/translate/(?P<dir_path>(.*/)*)?/?',
+        VFolderTPTranslateView.as_view(),
+        name='vfolder-pootle-tp-translate'),
```

Comment:
This doesn't follow the urlpattern names agreements. Use `pootle-virtualfolder-translate` instead.